src/system/data-export.md

```diff
@@ -8,44 +8,44 @@ Special characters—such as the equal sign, greater and less than symbols, sing
 
```

Comment:
suggestion:   below is the special character paragraph copied from the Import topic.  use same formatting/wording for consistency?

Special characters (such as the equal sign, greater and less than symbols, single and double quotes, backslash, pipe, and ampersand symbols) can cause problems during data transfer. To ensure that such special characters are correctly interpreted, they can be marked as an _escape sequence_. For example, if the data includes a string of text such as `code=”str”`, `code=”str2”`, choosing to enclose the text in double quotes ensures that the original double quotes are understood to be part of the data. When the system encounters a double set of double quotes, it understands that the outer set of double quotes is enclosing the actual data.